### DISCUSSION TITLE
List of Dynamic Tables Dataset Available

### DISCUSSION THREAD

**Discussion by satria-aji** [dzulfiqarfr/bpsr#6] [Q&A] [answered]
Hi, i want to ask a very basic question which I havent found the answer. Is there any way to see the full documentation of what dataset from dynamic table that are available from the API (in the form of list of dataset and its ID available, compared to the bps_dataset() that only shows few)?

**Comment by dzulfiqarfr** [comment, marked as answer]
Hello. You can use `bps_dataset()` and set `page` to `Inf` to get the complete list of datasets, as follows.

```
bps_dataset(page = Inf)
```

You can read more about the function on [the package's documentation website](https://dzulfiqarfr.github.io/bpsr/reference/bps_dataset.html).

Hope this helps!
